automated/lib/py_util_lib.py

```diff
@@ -0,0 +1,45 @@
+"""Shared Python 3 utility code.
```

Comment:
We already have `automated/lib/py_test_lib.py` for the same purpose. Is there a reason to add a new one?

Reply:
Sounds good to me.